### ISSUE TITLE
Bad database downloads with v6.1.0

### ISSUE THREAD

**Issue by dominicraf** [extremeshok/clamav-unofficial-sigs#241]
Using the latest version v6.1.0 I see messages thus:
```
Clamscan reports Sanesecurity jurlbl.ndb database integrity tested BAD
Removed invalid database: /var/db/clamav-unofficial-sigs/dbs-ss/jurlbl.ndb
Testing updated Sanesecurity database file: blurl.ndb
Clamscan reports Sanesecurity blurl.ndb database integrity tested BAD
Removed invalid database: /var/db/clamav-unofficial-sigs/dbs-ss/blurl.ndb
Testing updated Sanesecurity database file: winnow_phish_complete_url.ndb
Clamscan reports Sanesecurity winnow_phish_complete_url.ndb database integrity tested BAD
Removed invalid database: /var/db/clamav-unofficial-sigs/dbs-ss/winnow_phish_complete_url.ndb
Testing updated Sanesecurity database file: bofhland_phishing_URL.ndb
Clamscan reports Sanesecurity bofhland_phishing_URL.ndb database integrity tested BAD
Removed invalid database: /var/db/clamav-unofficial-sigs/dbs-ss/bofhland_phishing_URL.ndb
Testing updated Sanesecurity database file: phishtank.ndb
Clamscan reports Sanesecurity phishtank.ndb database integrity tested BAD
Removed invalid database: /var/db/clamav-unofficial-sigs/dbs-ss/phishtank.ndb
Testing updated Sanesecurity database file: porcupine.hsb
Clamscan reports Sanesecurity porcupine.hsb database integrity tested BAD
Removed invalid database: /var/db/clamav-unofficial-sigs/dbs-ss/porcupine.hsb
```
Reverting to the previous version the downloads all test good.

**Comment by extremeshok** [comment]
can you please post your clamav version and os

**Comment by dominicraf** [comment]
ClamAV 0.100.3
Ubuntu 18.04.3

**Comment by extremeshok** [comment]
Im not having any issues, this is my one test server.

``
root@ununtu-server:/# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 19.04
Release:	19.04
Codename:	disco
root@ununtu-server:/# clamscan -V
ClamAV 0.100.3/25556/Thu Aug 29 08:25:39 2019

``

``
root@ununtu-server:/# /usr/local/sbin/clamav-unofficial-sigs.sh
################################################################################
 eXtremeSHOK.com ClamAV Unofficial Signature Updater
 Version: v6.1.0 (2019-08-27)
 Required Configuration Version: v76
 Copyright (c) Adrian Jon Kriel :: admin@extremeshok.com
################################################################################
Loading config: /etc/clamav-unofficial-sigs/master.conf
Loading config: /etc/clamav-unofficial-sigs/os.ubuntu.conf
Loading config: /etc/clamav-unofficial-sigs/user.conf
===================
Preparing Databases
===================
==================================================
Sanesecurity Database & GPG Signature File Updates
==================================================
Checking for Sanesecurity updates...
Sanesecurity mirror site used: rsync-mirror.rollernet.us 208.79.241.67
receiving file list ... 90 files to consider
blurl.ndb
      4,864,136 100%    3.69MB/s    0:00:01 (xfr#1, to-chk=85/90)
blurl.ndb.sig
             72 100%    0.24kB/s    0:00:00 (xfr#2, to-chk=84/90)
bofhland_phishing_URL.ndb
          3,544 100%   11.65kB/s    0:00:00 (xfr#3, to-chk=77/90)
bofhland_phishing_URL.ndb.sig
             72 100%    0.24kB/s    0:00:00 (xfr#4, to-chk=76/90)
junk.ndb
      7,275,210 100%   20.23MB/s    0:00:00 (xfr#5, to-chk=65/90)
junk.ndb.sig
             72 100%    0.20kB/s    0:00:00 (xfr#6, to-chk=64/90)
jurlbl.ndb
      1,369,691 100%    3.59MB/s    0:00:00 (xfr#7, to-chk=63/90)
jurlbl.ndb.sig
             72 100%    0.19kB/s    0:00:00 (xfr#8, to-chk=62/90)
jurlbla.ndb
        169,030 100%  402.61kB/s    0:00:00 (xfr#9, to-chk=61/90)
jurlbla.ndb.sig
             72 100%    0.17kB/s    0:00:00 (xfr#10, to-chk=60/90)
phish.ndb
      4,095,787 100%    8.98MB/s    0:00:00 (xfr#11, to-chk=47/90)
phish.ndb.sig
             72 100%    0.16kB/s    0:00:00 (xfr#12, to-chk=46/90)
phishtank.ndb
      1,698,226 100%    2.81MB/s    0:00:00 (xfr#13, to-chk=45/90)
phishtank.ndb.sig
             72 100%    0.12kB/s    0:00:00 (xfr#14, to-chk=44/90)
porcupine.hsb
         38,565 100%   64.71kB/s    0:00:00 (xfr#15, to-chk=43/90)
porcupine.hsb.sig
             72 100%    0.12kB/s    0:00:00 (xfr#16, to-chk=42/90)
porcupine.ndb
        561,559 100%  932.65kB/s    0:00:00 (xfr#17, to-chk=41/90)
porcupine.ndb.sig
             72 100%    0.12kB/s    0:00:00 (xfr#18, to-chk=40/90)
rogue.hdb
        438,405 100%  723.19kB/s    0:00:00 (xfr#19, to-chk=39/90)
rogue.hdb.sig
             72 100%    0.12kB/s    0:00:00 (xfr#20, to-chk=38/90)
scam.ndb
      1,907,696 100%    3.02MB/s    0:00:00 (xfr#21, to-chk=35/90)
scam.ndb.sig
             72 100%    0.12kB/s    0:00:00 (xfr#22, to-chk=34/90)
spamimg.hdb
         18,632 100%   30.07kB/s    0:00:00 (xfr#23, to-chk=23/90)
spamimg.hdb.sig
             72 100%    0.12kB/s    0:00:00 (xfr#24, to-chk=22/90)
winnow_phish_complete_url.ndb
          6,577 100%    8.26kB/s    0:00:00 (xfr#25, to-chk=3/90)
winnow_phish_complete_url.ndb.sig
             72 100%    0.09kB/s    0:00:00 (xfr#26, to-chk=2/90)
Testing updated Sanesecurity database file: junk.ndb
Sanesecurity GPG Signature tested good on junk.ndb database
Clamscan reports Sanesecurity junk.ndb database integrity tested good
Successfully updated Sanesecurity production database file: junk.ndb
Testing updated Sanesecurity database file: jurlbl.ndb
Sanesecurity GPG Signature tested good on jurlbl.ndb database
Clamscan reports Sanesecurity jurlbl.ndb database integrity tested good
Successfully updated Sanesecurity production database file: jurlbl.ndb
Testing updated Sanesecurity database file: phish.ndb
Sanesecurity GPG Signature tested good on phish.ndb database
Clamscan reports Sanesecurity phish.ndb database integrity tested good
Successfully updated Sanesecurity production database file: phish.ndb
Testing updated Sanesecurity database file: rogue.hdb
Sanesecurity GPG Signature tested good on rogue.hdb database
Clamscan reports Sanesecurity rogue.hdb database integrity tested good
Successfully updated Sanesecurity production database file: rogue.hdb
Testing updated Sanesecurity database file: scam.ndb
Sanesecurity GPG Signature tested good on scam.ndb database
Clamscan reports Sanesecurity scam.ndb database integrity tested good
Successfully updated Sanesecurity production database file: scam.ndb
Testing updated Sanesecurity database file: spamimg.hdb
Sanesecurity GPG Signature tested good on spamimg.hdb database
Clamscan reports Sanesecurity spamimg.hdb database integrity tested good
Successfully updated Sanesecurity production database file: spamimg.hdb
Testing updated Sanesecurity database file: blurl.ndb
Sanesecurity GPG Signature tested good on blurl.ndb database
Clamscan reports Sanesecurity blurl.ndb database integrity tested good
Successfully updated Sanesecurity production database file: blurl.ndb
Testing updated Sanesecurity database file: jurlbla.ndb
Sanesecurity GPG Signature tested good on jurlbla.ndb database
Clamscan reports Sanesecurity jurlbla.ndb database integrity tested good
Successfully updated Sanesecurity production database file: jurlbla.ndb
Testing updated Sanesecurity database file: winnow_phish_complete_url.ndb
Sanesecurity GPG Signature tested good on winnow_phish_complete_url.ndb database
Clamscan reports Sanesecurity winnow_phish_complete_url.ndb database integrity tested good
Successfully updated Sanesecurity production database file: winnow_phish_complete_url.ndb
Testing updated Sanesecurity database file: bofhland_phishing_URL.ndb
Sanesecurity GPG Signature tested good on bofhland_phishing_URL.ndb database
Clamscan reports Sanesecurity bofhland_phishing_URL.ndb database integrity tested good
Successfully updated Sanesecurity production database file: bofhland_phishing_URL.ndb
Testing updated Sanesecurity database file: porcupine.ndb
Sanesecurity GPG Signature tested good on porcupine.ndb database
Clamscan reports Sanesecurity porcupine.ndb database integrity tested good
Successfully updated Sanesecurity production database file: porcupine.ndb
Testing updated Sanesecurity database file: phishtank.ndb
Sanesecurity GPG Signature tested good on phishtank.ndb database
Clamscan reports Sanesecurity phishtank.ndb database integrity tested good
Successfully updated Sanesecurity production database file: phishtank.ndb
Testing updated Sanesecurity database file: porcupine.hsb
Sanesecurity GPG Signature tested good on porcupine.hsb database
Clamscan reports Sanesecurity porcupine.hsb database integrity tested good
Successfully updated Sanesecurity production database file: porcupine.hsb
========================================
linuxmalwaredetect Database File Updates
========================================
Checking for linuxmalwaredetect updates...
Checking for updated linuxmalwaredetect database file: rfxn.ndb
2019-08-29 13:25:31 URL:http://cdn.rfxn.com/downloads/rfxn.ndb [451444/451444] -> "rfxn.ndb" [1]
No updated linuxmalwaredetect rfxn.ndb database file found
---
Checking for updated linuxmalwaredetect database file: rfxn.hdb
2019-08-29 13:25:31 URL:http://cdn.rfxn.com/downloads/rfxn.hdb [851688/851688] -> "rfxn.hdb" [1]
Testing updated linuxmalwaredetect database file: rfxn.hdb
Clamscan reports linuxmalwaredetect rfxn.hdb database integrity tested good
Successfully updated linuxmalwaredetect production database file: rfxn.hdb
================================================================================
================================================================================
No whitelist signature changes found in my-whitelist.ign2
=================================
Setting permissions and ownership
=================================
==============================================
Update(s) detected, reloading ClamAV databases
==============================================
=========================
ClamAV databases Reloaded
=========================
----------------------------------------------------------------------------
Issue tracker : https://github.com/extremeshok/clamav-unofficial-sigs/issues
----------------------------------------------------------------------------
##############################################
      Powered By https://eXtremeSHOK.com      
##############################################

``

**Comment by extremeshok** [comment]
can you retest please, havnt had any other reports. All my centos6/7 boxen are working.

**Comment by extremeshok** [comment]
these issues are fixed in the latest dev, and 6.1.1
